cssom-view/Overview.bs

```diff
@@ -896,13 +897,35 @@ screen area</a>. <!--fingerprint-->
 The <dfn attribute for=Screen>height</dfn> attribute must return the height of the <a>Web-exposed
 screen area</a>. <!--fingerprint-->
 
-The <dfn attribute for=Screen>colorDepth</dfn> attribute must return 24.
+The <dfn attribute for=Screen>colorDepth</dfn> and <dfn attribute
+for=Screen>pixelDepth</dfn> attributes should return the number of bits
+allocated to colors for a pixel in the output device, excluding the alpha
+channel. If the user agent is not able to return the number of bits used by the
+output device, it should return the closest estimation such as, for example, the
+number of bits used by the frame buffer sent to the display or any internal
+representation that would be the closest to the value the output device would
+use. The user agent must return the same value for these attributes and the
```

Comment:
Same value seems wrong. The color media query is per component, whereas here is all components together. There's also this complication:

> If different color components are represented by different number of bits, the smallest number is used.

I suppose an integer-divide by 3 should give the same value as the color media query?